packages/ap-codegen/alliance_platform/codegen/printer.py

```diff
@@ -190,14 +215,34 @@ def _push_node(self, node: NodeLike):
         finally:
             self.node_stack.pop()
 
+    def _format_jsx_attribute_string(self, value: str):
+        """Format a string for use in JSX attribute
+
+        If string would need to be escaped, that escaped version  will be wrapped in curly braces to make it an
+        expression.
+
+        In JSX, this isn't valid: myString="this is a \"quoted\" string"
+
+        That particular example could be myString='this is a "quoted" string', but for simpler implementation
+        we'll just use the curly braces which handles other cases like myString=" a 'mixed' \"quoted\" string"
+        """
+        formatted_value = self._format_literal(value)
+        if f'"{value}"' != formatted_value:
+            # If escaping is needed, use a JSX expression. For example, if the string is "Hello <World>" then
+            # this will be printed as {"Hello <World>"}
+            return f"{{{formatted_value}}}"
+        return formatted_value
```

Comment:
this change was directly related to the XSS fixes, but fixes a bug that could happen in dev when generating a debug component tree(nothing broke, but you got a warning in the console)